src/graphs.py

```diff
@@ -0,0 +1,222 @@
+from matplotlib import pyplot as plt
+from matplotlib import tri as tri
+from matplotlib import cm
+import scienceplots
+import numpy as np
+import os
+
+
+
+class GraphManager():
+    def __init__(self):
+        plt.style.use('science')
+    
+
+    def draw(self):
+        plt.show()
+        plt.close()
+
+    
+    def save_png(self, folder, file_name):
+        dir_path = os.path.dirname(os.path.dirname(__file__))
+        file_path = os.path.join(os.path.sep, dir_path, folder, file_name)
+        plt.savefig(file_path)
+        plt.close()
```

Comment:
I think `plt.savefig()` can handle being passed a relative or absolute path, so this could be simplified as follows:
```suggestion
    def save_png(self, file_path):
        plt.savefig(file_path)
        plt.close()
```